pages/home.md

```diff
@@ -14,14 +14,14 @@ permalink: /
         />
       </figure>
       <p class="subtitle has-text-centered is-size-4 is-size-3-tablet is-size-2-desktop">
-      Using Web standards to
-      let people control their data, and choose the
-      applications and services to use with it.
+      Developing Web standards that
+      enables people to control their data. 
```

Comment:
I think "control their data" would encapsulate "choose the applications and services to use with it."  

Alternatively, we could reword the second half to: ", including which applications/services can access their data."

Reply:
I am missing the word "choice" though; very important to Solid.

Reply:
👍  - will try to work that back in the next patch

Reply:
Gave it a shot - but not sure if what I have is quite right.

Reply:
Resolving this since below comment address this further.